open-metadata-implementation/access-services/asset-lineage/asset-lineage-server/pom.xml

```diff
@@ -75,6 +75,13 @@
             <scope>compile</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-core</artifactId>
+            <version>5.2.1.RELEASE</version>
```

Comment:
The version should be removed here (and don't use hardcoded values). The version should be picked up automatically from top level pom's dependencyManagement